### DISCUSSION TITLE
Custom sx utility props

### DISCUSSION THREAD

**Discussion by ashtonlance** [mantinedev/mantine#2301] [Q&A]
I want to be able to use utility aliases in my `sx` prop, such as `px` for paddingLeft & paddingRight and `py` for paddingTop & paddingBottom.

I've seen this pattern in lots of other libraries, is there a way to set this up within Mantine or Emotion?

As an example from MUI: https://mui.com/system/getting-started/the-sx-prop/#spacing

**Comment by rtivital** [comment]
No, currently we do not provide such feature
